### PR TITLE
[AMSFT102020]issaGruber

### PR DESCRIPTION
**Pull request by GruberI**
mostly finished still some styling needed

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -12,7 +11,74 @@ app.set('views', __dirname + '/views');
...
+    .then(data => spotifyApi.setAccessToken(data.body['access_token']))
+    .catch(error => console.log('Something went wrong when retrieving an access token', error));
 
 // Our routes go here:
+app.get('/', (req, res) => {
+    res.render('index');
+  });
+
+app.get('/artist-search', (req, res) => {
+    // console.log(req.query)
```
Comment (LloydC) [inline app.js:31]: Remember to remove `console.logs` when you're done building your application.

File: app.js [hunk 2]
```diff
@@ -12,7 +11,74 @@ app.set('views', __dirname + '/views');
...
+            const { items } = data.body;
+            
+            res.render("tracks", { tracks: items});
+      })
+      .catch(err => console.log('The error while seaching tracks occured: ', err));
+
+  })
+
+
+// spotifyApi
```
Comment (LloydC) [inline app.js:73]: Also remove commented out code to keep your file size minimal.

File: views/albums.hbs [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<body class="albumsBody">
+    <div class="container">
+        <ul class="albumsUl">
```
Comment (LloydC) [inline views/albums.hbs:3]: Only `<li>` can be nested directly after a `<ul>` element.

You can nest a `<div>` inside the  nested`<li>`
```
<ul class="albumsUl">
  {{#each albums}}
  <li>
            <div class="albumsCard">
                <img src={{this.images.2.url}} alt="albumcover" class="albumsImg">
                <div class="albumsText">
                    <span class="albumsLi">{{this.name}}</span>
                    <br>
                    <button class="albumsButton"><a href="/tracks/{{this.id}}" class="page3a">View Tracks</a></button>
                    </div>
                 </div>
                </li>
            {{/each}}
        </ul>

File: views/artist-search-results.hbs [hunk 4]
```diff
@@ -0,0 +1,25 @@
+<body id="page2body">
```
Comment (LloydC) [inline views/artist-search-results.hbs:1]: You don't need the `<body>` element here, as everything here will render inside the body element of `layout.hbs`

File: views/tracks.hbs [hunk 5]
```diff
@@ -0,0 +1,33 @@
+<body class="tracksBody">
```
Comment (LloydC) [inline views/tracks.hbs:1]: Same comment as above

File: views/artist-search-results.hbs [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+  <div class="page2container">
```
Comment (LloydC) [inline views/artist-search-results.hbs:2]: I recommend not using numbers when you define class names or ids. You can use '-' or camel case when dealing with multiple words (i.e: page-container or pageToContainer)

**Comment by GruberI** [comment]
Thank you! I do that a lot, I will change that.

**Pull request closed without merging**